### PR TITLE
Add issymmetric name change.

### PR DESCRIPTION
**Pull request by pkofod**
Added readme entry, name change, and two small tests making sure `issymmetric` works.

### CODE REVIEW

**Comment by pkofod** [comment]
Hm, it fails. On my v0.4 install it works.

File: src/Compat.jl [hunk 1]
```diff
@@ -868,3 +868,8 @@ module LinAlg
 end
 
 end # module
+
+if VERSION < v"0.5.0-dev+2915"
+    const issymmetric = issym
+    export issymmetric
+end
```
Comment (yuyichao) [inline src/Compat.jl]: This needs to be in the module

File: README.md [hunk 2]
```diff
@@ -146,6 +146,8 @@ Currently, the `@compat` macro supports the following syntaxes:
 
 * `Compat.LinAlg.checksquare` [#14601](https://github.com/JuliaLang/julia/pull/14601)
 
+* `issym` is not `issymmetric` [#15192](https://github.com/JuliaLang/julia/pull/15192)
```
Comment (tkelman) [inline README.md]: is now

**Comment by pkofod** [comment]
Fixed.

**Pull request merged**